src/components/views/auth/CountryDropdown.js

```diff
@@ -45,17 +46,24 @@ export default class CountryDropdown extends React.Component {
         this._onOptionChange = this._onOptionChange.bind(this);
         this._getShortOption = this._getShortOption.bind(this);
 
+        let defaultCountry = COUNTRIES[0];
+        if (SdkConfig.get()["defaultCountryCode"]) {
+            const country = COUNTRIES.find(c => c.iso2 === SdkConfig.get()["defaultCountryCode"]);
```

Comment:
does the config need a toUppercase or something to make it a bit more robust?

Reply:
Wouldn't hurt, will add.